frontend/src/components/common/Scenarios/ScenarioPlot.js

```diff
@@ -44,7 +46,7 @@ const ScenarioPlot = ({ scenarioData, scenarioSelectedRuptures, extra }) => {
           type: "scatter",
           mode: "lines",
           line: { color: deafultColours[colourCounter % 10] },
-          name: `Scenario ${curRup} [16, 50, 84 %ile]`,
```

Comment:
It was me who didn't remove `Scenario` but like every other plot, I think we can safely remove the `Scenario`
Like Claudio said the other day, users are already on the Scenario page and no other plots(Hazard Curve, UHs and GMS), we do not say `Hazard Curve - something`.